src/components/shared-settings/useSettingsUI.tsx

```diff
@@ -56,6 +60,11 @@ export function SettingsUI<T>(props: SettingsUIProps<T>) {
             )
     }
 }
+
+const useStyles2 = makeStyles({
```

Comment:
Merge this style into the previous one? Or another name?

Reply:
renamed